docker/base/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER edxops
 USER root
 RUN apt-get update \
       && apt-get install -y sudo python-dev python-pip libmysqlclient-dev \
-         libssl-dev libffi-dev  \
+         libssl-dev libffi-dev xvfb iceweasel  \
```

Comment:
Iceweasel is Firefox on Debian, the apt-get install comes w/ a firefox binary as well as iceweasel